plugins/storage/fs/src/hopeit/fs_storage/__init__.py

```diff
@@ -131,6 +131,54 @@ async def store(self, key: str, value: DataObject) -> str:
             os.makedirs(path.resolve().as_posix(), exist_ok=True)
         return await self._save_file(payload_str, path=path, file_name=key + SUFFIX)
 
+    async def store_file(self, file_name: str, value: io.BytesIO) -> str:
+        """
+        Stores a file-like object.
+
+        :param file_name: str
+        :param value: io.BytesIO, the file-like object to store
+        :return: str file location
+        """
+        path = self.path
+        partition_key = ""
+        if self.partition_dateformat:
+            partition_key = get_partition_key(None, self.partition_dateformat)
```

Comment:
Instead of calling same function wit None, make an specific function to call and avoid the if statement inside get_partition_key